src/transport/SessionManager.cpp

```diff
@@ -414,6 +414,27 @@ CHIP_ERROR SessionManager::InjectPaseSessionWithTestKey(SessionHolder & sessionH
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR SessionManager::InjectCaseSessionWithTestKey(SessionHolder & sessionHolder, uint16_t localSessionId, uint16_t peerSessionId,
```

Comment:
Suggest adding a FabricIndex to make clear that it has to be *the same* for both peers. We could as well set the fabric index in the underlying secure session

Reply:
There is a `FabricIndex` argument